src/OrbitBase/include/OrbitBase/TaskGroup.h

```diff
@@ -47,7 +48,7 @@ class TaskGroup {
   }
 
  private:
-  orbit_base::Executor* executor_ = nullptr;
+  std::shared_ptr<orbit_base::Executor> executor_ = nullptr;
```

Comment:
I would prefer to keep this a non-owning pointer.

Reply:
Done, thanks.